CRM/Core/BAO/Cache.php

```diff
@@ -64,8 +64,13 @@ class CRM_Core_BAO_Cache extends CRM_Core_DAO_Cache {
    *
    * @return object
    *   The data if present in cache, else null
+   * @deprecated
    */
   public static function &getItem($group, $path, $componentID = NULL) {
+    if ($adapter = CRM_Utils_Constant::value('CIVICRM_BAO_CACHE_ADAPTER')) {
```

Comment:
From an IDE POV this construct

if (($adapter = CRM_Utils_Constant::value('CIVICRM_BAO_CACHE_ADAPTER')) !== FALSE) {

doesn't give warnings. I believe the case for it is also that it disambiguates between '= means =' & '= is a bug because I forgot the second one'

Reply:
@eileenmcnaughton I've pushed up a revision to use that notation. Notes:

* FWIW, my IDE doesn't complain about either the old or new notation. (PHPStorm 2016.2.2.)
* `CRM_Utils_Constant::value()` returns a default of `NULL`, so I used that instead of `FALSE`.